modules/rtdModule/index.ts

```diff
@@ -166,7 +166,8 @@ export const setBidRequestsData = timedAuctionHook('rtd', function setBidRequest
 
   relevantSubModules.forEach(sm => {
     const fpdGuard = guardOrtb2Fragments(reqBidsConfigObj.ortb2Fragments || {}, activityParams(MODULE_TYPE_RTD, sm.name));
-    sm.getBidRequestData({...reqBidsConfigObj, ortb2Fragments: fpdGuard}, onGetBidRequestDataCallback.bind(sm), sm.config, _userConsent, timeout);
+    reqBidsConfigObj.ortb2Fragments = mergeDeep({}, reqBidsConfigObj.ortb2Fragments || {}, fpdGuard);
```

Comment:
unfortunately this reintroduces a version of the problem that https://github.com/prebid/Prebid.js/pull/14108 was trying to address, the global `ortb2Fragments` is affected by what applies to RTD modules.